torch/csrc/jit/passes/graph_fuser.cpp

```diff
@@ -905,9 +905,14 @@ struct GraphFuser {
 } // anonymous namespace
 
 void FuseGraph(std::shared_ptr<Graph>& graph) {
+  // NYI on Windows
+  #ifndef _WIN32
+
   GraphFuser(graph->block()).run();
   // After FuseGraph some common subexpressions may come back
   EliminateCommonSubexpression(graph);
+
+  #endif
```

Comment:
Why is that needed again? It would be better to make the pass a no-op if the fuser doesn't work.

Reply:
Isn't it currently making the pass a no-op?

I don't have the full context on why fuser doesn't work on windows.

Reply:
Never mind. I confused this code with that in `graph_executor.cpp`, which looks exactly the same now.